### PR TITLE
Submission for .NET MAUI Beautiful UI Challenge

### PR DESCRIPTION
**Pull request by behl1anmol**
Submitting a Todo app created using .NET MAUI Beautiful UI Challenge

### CODE REVIEW

File: src/Snppts/Snippets/TodoMe.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    public bool ContainsUWPSample => false;
+
+    public Sdk BuiltWith => Sdk.Maui;
+
+    public IAmAnAuthor AuthorInfo => new AnmolBehl();
+
+    public IEnumerable<Uri> ImageUris => new List<Uri>
+    {
+        new Uri("https://github.com/behl1anmol/Todo.me/blob/master/Screenshots/Maui_app_todo.gif"),
```
Comment (jfversluis) [inline src/Snppts/Snippets/TodoMe.cs]: Can you change this one to point directly to the image instead of the GitHub page?  Thanks!

Comment (behl1anmol) [reply src/Snppts/Snippets/TodoMe.cs]: I have made the necessary changes. 
Regards,
Anmol

**Review by jfversluis — APPROVED** [review]
Great stuff looking great, thanks for your submission Anmol!

**Comment by behl1anmol** [comment]
> Great stuff looking great, thanks for your submission Anmol!

Thank You, Gerald
Looking forward to making more such valuable contributions.

**Pull request merged**